api/src/pcapi/core/users/external/models.py

```diff
@@ -25,7 +25,7 @@ class UserAttributes:
     is_active: bool  # Added for Zendesk
     is_beneficiary: bool
     is_current_beneficiary: bool  # Beneficiary with a non-expired remaining credit
-    is_former_beneficiary: bool  # Beneficiary which credit is expired or spent
+    is_former_beneficiary: bool  # Beneficiary which last possible credit is definitely expired or spent
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** 
```suggestion
    is_former_beneficiary: bool  # Beneficiary whose last possible credit is definitely expired or spent
```

Reply:
bien vu :)